foundry/pixi/containers/ruler.d.ts

```diff
@@ -1,167 +1,184 @@
-declare interface RulerData {
-  _state: number;
-  class: string;
-  destination: PIXI.Point;
-  name: string;
-  waypoints: [PIXI.Point];
-}
-
 /**
  * The Ruler - used to measure distances and trigger movements
- * @param user - The User for whom to construct the Ruler instance
+ * @param user  - The User for whom to construct the Ruler instance
+ * @param color - The color for the ruler
+ *                (Default value: The user's color or `0x42f4e2`)
  */
 declare class Ruler extends PIXI.Container {
+  constructor(user?: User, { color }?: { color?: number | null });
+
   /**
-   * Track the current measurement state
+   * Record the User which this Ruler references
    */
-  public _state: number;
+  user: User;
 
   /**
-   * Is the Ruler being actively used to measure distance?
+   * The ruler name - used to differentiate between players
    */
-  public readonly active: boolean;
+  name: string;
 
   /**
    * The ruler color - by default the color of the active user
    */
-  public color: number | null;
+  color: number | null;
+
+  /**
+   * This Array tracks individual waypoints along the ruler's measured path.
+   * The first waypoint is always the origin of the route.
+   */
+  waypoints: PIXI.Point[];
 
   /**
    * The current destination point at the end of the measurement
    */
-  public destination: PIXI.Point;
+  destination: PIXI.Point;
```

Comment:
Seems like this can be `null`:
```js
    /**
     * The current destination point at the end of the measurement
     * @type {PIXI.Point}
     */
    this.destination = null;
```